src/pages/en/guides/testing.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Testing
+description: An intro to testing in Astro
+i18nReady: false
+---
+
+## Playwright
+
+In their own words, Playwright is a testing framework that let's you write end-to-end testing for modern web apps. You can do this cross browser, cross platform, and across multiple languages, but for our purposes the scope is clearly Javascript/Typescript. 
+
+### Quickstart
+
+The fastest way to get started is to use `yarn create playwright` within your Astro project. This will guide you through CLI steps to choose Javascript/Typescript, name your test folder, and add an optional Github Actions workflow. 
```

Comment:
(followup to below) if the package manager tabs work here, lets remove the `yarn` use here.  Would "is to use `create playwright`" still make sense here?